### DISCUSSION TITLE
stream_port

### DISCUSSION THREAD

**Discussion by dia-de-muertos** [Motion-Project/motionplus#22] [General]
Hi,
can't find a option to configure the port for the stream (stream_port in 'legacy' motion).
Is it not possible any more?

**Comment by Mr-Dave** [comment]
There is only one port for MotionPlus and it supports both the webcontrol as well as the streams.

**Reply by dia-de-muertos** [reply]
OK, thanks for the clarification.
This means I can access different streams via different URLs and not ports (never enabled webcontrol yet)?

**Comment by Mr-Dave** [comment]
Correct.  The cameras are at different URLs.  

The webcontrol is also at a different URL on the same port but the configuration now allows essentially everything to be disabled if it isn't desired.

User created webcontrol pages are also an option.

**Comment by dia-de-muertos** [comment]
Ok, thanks for the fast answer and clarification.
Will test later this week.

**Comment by seventhsite** [comment]
Is there any documentation about it?
By indirect method, I found the URL of the stream - http://IP:8081/1/mjpg
How can I find the URL of a static image?

**Reply by gabbymouth** [reply]
For static image it would be http://ip:8081/1/static 
There are examples here: https://motion-project.github.io/motionplus_examples.html#webcontrol_pages
